tests/implementor/test_table_simulation_configuration.py

```diff
@@ -17,28 +17,21 @@ def setup_method(self):
 
     @pytest.mark.parametrize(
         "init_dict",
-        [
-            {  # token does not exists
-                "token": "00000000-0000-0000-0000-000000000000",
-            },
-            {},  # token is missing
-        ],
+        [],
```

Comment:
```suggestion
        [{}],
```

Reply:
Creating a SimulationConfiguration without any arguments is fine now :)